### DISCUSSION TITLE
"edit Link - length"

### DISCUSSION THREAD

**Discussion by sangbosim** [pyswmm/pyswmm#403] [Q&A] [answered]
Dear all,

With the help of Pyswmm, I am modifying the link connection of a model. However, I am asking because I don't think there is a function that can read and modify the length of the link no matter how much I look for.
Is there no query in pyswmm that can read and modify the length of the link?

Could anyone tell me how to modify the length of the link?

regard.

**Comment by bemcdonnell** [comment, marked as answer]
@sangbosim great question! I believe this would be a useful addition to PySWMM.  You are correct - this is currently an unsupported feature.  Would you be willing to help us integrate it if we walk you through the various steps?

**Reply by sangbosim** [reply]
Thank you for your answer.
If you let me know, I will proceed with the work as much as I can.
Additionally, is there any other tool to read the length of the link?

Regard.

**Reply by bemcdonnell** [reply]
Check out SWMMIO python package :-)

**Reply by sangbosim** [reply]
Thank you :)
